pallets/dao/src/lib.rs

```diff
@@ -421,28 +441,56 @@ pub mod pallet {
 
 	// *** Helper functions *** //
 	impl<T:Config> Pallet<T> {
-		pub fn new_org(from_initiator: &T::AccountId, org_name: &[u8]) -> Result<(), DispatchError> {
-
-			let mut org = <Pallet<T>>::organization(org_name);
-			org.push(from_initiator.clone());
+		pub fn new_org(
+			from_initiator: &T::AccountId,
+			name: &[u8],
+			description: &[u8],
+			vision: &[u8],
+		) -> Result<T::Hash, DispatchError> {
+
+			let dao = Dao::<T> {
+				name: name.to_vec(),
+				description: description.to_vec(),
+				owner: from_initiator.clone(),
+				vision: vision.to_vec(),
+				created_time: 0,
```

Comment:
You can use **block number** as create_time for now
let current_block = <frame_system::Pallet<T>>::block_number();